vint/linting/policy/prohibit_implicit_scope_variable.py

```diff
@@ -22,10 +22,18 @@ def is_valid(self, identifier, lint_context):
 
         scope_plugin = lint_context['plugins']['scope']
         explicity = scope_plugin.get_explicity_of_scope_visibility(identifier)
+        is_autoload = scope_plugin.is_autoload_identifier(identifier)
 
-        self._make_description(identifier, scope_plugin)
+        config_dict = lint_context['config']
+        suppress_autoload = self.get_policy_option(config_dict, 'suppress_autoload', False)
```

Comment:
Right. Also it is actually better to return all policy options in a dict (back to `is_valid`) because if there are more options, we can easily access them without additional function calls.